consensus/validator_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package consensus
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/vechain/thor/v2/block"
+	"github.com/vechain/thor/v2/chain"
+	"github.com/vechain/thor/v2/genesis"
+	"github.com/vechain/thor/v2/muxdb"
+	"github.com/vechain/thor/v2/state"
+	"github.com/vechain/thor/v2/thor"
+	"github.com/vechain/thor/v2/tx"
+)
+
+func TestValidateBlockBody(t *testing.T) {
```

Comment:
Might be good to add tests for the header.BaseFee before galactica and Invalid gas limit ?

Reply:
We already have baseFee before galactica [here](https://github.com/vechain/thor/blob/feat/base-fee/consensus/consensus_test.go#L450), and for gas limit [here](https://github.com/vechain/thor/blob/feat/base-fee/consensus/consensus_test.go#L352)

Reply:
I agree, I'm thinking If someone changes the validator code, for example [here](https://github.com/vechain/thor/pull/933/files/47454ff4a3839d8d159e2eea887235348cdcf2fa#diff-a59b0fd7b9784fd3e9169cc57ebefb04b4378bac1d7bcc3d33b414ca540c12b0R157-R159) would that change be captured in the consensus tests ?